src/Abstractions/NexusMods.Abstractions.NexusWebApi/Types/V2/GameId.cs

```diff
@@ -23,7 +23,7 @@ public static GameId FromGameDomain(GameDomain domain)
     {
         return domain.Value switch
         {
-            "stardewvalley" => (GameId)1704,
+            "stardewvalley" => (GameId)1303,
```

Comment:
What is this about?

Reply:
It was an accident when I introduced the temporary mapping, during copy-pasting. My branch which throws out as much V1 as possible does proper resolution via the website.

I noticed this bug this morning, and @erri120 ran into it later too.